### PR TITLE
MSSQL: CloudWatch properties exposed

### PR DESCRIPTION
**Pull request by zucchinidev**
[#185202654](https://www.pivotaltracker.com/story/show/185202654)

Which other branches should this be merged with (if any)?

None

### CODE REVIEW

File: reference/aws-mssql.html.md.erb [hunk 1]
```diff
@@ -492,6 +492,70 @@ provision only:
...
+    <tr>
+      <td><code>cloudwatch_log_groups_kms_key_id</code></td>
+      <td>String</td>
+      <td>
+        Log group data is always encrypted in CloudWatch Logs. By default, CloudWatch Logs uses
+        server-side encryption for the log data at rest.
+        As an alternative, you can use AWS Key Management Service (AWS KMS) for this encryption.
+        If you use AWS KMS, the encryption is done by using an AWS KMS customer-managed key.<br><br>
+        This property, if provided, sets the customer-managed key to use for encrypting the CloudWatch
+        log group created for the `agent` and `error` logs.
```
Comment (fnaranjo-vmw) [inline reference/aws-mssql.html.md.erb:552]:
```suggestion
        log group created for the <code>agent</code> and <code>error</code> logs.
```

File: reference/aws-mssql.html.md.erb [hunk 2]
```diff
@@ -492,6 +492,70 @@ provision only:
...
       <td>provision and update</td>
     </tr>
+    <tr>
+      <td><code>enable_export_agent_logs</code></td>
+      <td>Boolean</td>
+      <td>
+        If <code>true</code>, it enables the `agent` <code>cloud_watch_log_export</code>
+        on the RDS instance.
+        When activated, it creates an associated `agent` CloudWatch log group.
+        When deactivated, the associated `agebt` CloudWatch log group is deleted.
```
Comment (fnaranjo-vmw) [inline reference/aws-mssql.html.md.erb:502]:
```suggestion
        If <code>true</code>, it enables the <code>agent</code> <code>cloud_watch_log_export</code>
        on the RDS instance.
        When activated, it creates an associated <code>agent</code> CloudWatch log group.
        When deactivated, the associated <code>agent</code> CloudWatch log group is deleted.
```

**Review by fnaranjo-vmw — CHANGES_REQUESTED** [review]

**Comment by HenryBorys** [comment]
@zucchinidev please review @fnaranjo-vmw 's requested changes and resolve conflicts, and then I'll merge. Thanks!

**Comment by zucchinidev** [comment]
Thank you very much, @fnaranjo-vmw. Suggested changes accepted!
Thank you very much, @HenryBorys !! :rocket:

**Pull request merged**